isort/main.py

```diff
@@ -285,11 +285,16 @@ def create_parser():
                         help='Tells isort to ignore whitespace differences when --check-only is being used.')
     parser.add_argument('-y', '--apply', dest='apply', action='store_true',
                         help='Tells isort to apply changes recursively without asking')
+    parser.add_argument('--unsafe', dest='unsafe', action='store_true',
+                        help='Tells isort to look for files in standard library directories, etc. '
+                             'where it may not be safe to operate in')
```

Comment:
Rather than implement of a new command line option could we adapt this to be the default value for `--skip` (same as Black's approach)?

Reply:
I decided against that, because many current projects (all I currently have, more or less) have a custom `skip =` stanza in the configuration, so they wouldn't benefit from this at all.